docs/metrics.md

```diff
@@ -0,0 +1,466 @@
+# Metrics
+
+Since EventStoreDB version `v23.6.0`, some metrics in [prometheus format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format) are available on the `/metrics` endpoint. Promethus can be configured to scrape this endpoint directly. The metrics are configured in `telemetryconfig.json`. In the near future we hope to publish a sample grafana dashboard.
```

Comment:
```suggestion
EventStoreDB collects metrics in [prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats) , available on the `/metrics` endpoint. Promethus can be configured to scrape this endpoint directly. The metrics are configured in `telemetryconfig.json`. 
```
Resoning for the change: 
- the doc are versionend, so this will appear in 23.6 version and the following one. no need to mention "since"
- prefer to link to the online doc of prometheus ( so that _they_ can track that we use their format ) 
- remove "in the near future" : because it's a roadmap promise in the documentation .

Reply:
thanks Yves, most of those are my doing :)
- I agree about the link, thanks!
- I see what you mean about the promise, do you think it would be ok to say 'in the future'? im happy if you'd prefer to leave it out entirely
- I mentioned 23.6 because there is sort of some overlap with the older prometheus exporter and I wanted to make it clear somehow that this one is the new one (the old one is still documented).

Reply:
>  see what you mean about the promise, do you think it would be ok to say 'in the future'? im happy if you'd prefer to leave it out entirely

I'd leave it out .

> 23.6 : 
Let's amend the prometheus doc then ?
add a link to this , and make it clear it's about the Stats/gossip endopint